src/optimise.jl

```diff
@@ -56,7 +66,18 @@ function minimise(f, X::T, tol=1e-3) where {T}
 end
 
 
-function maximise(f, X::T, tol=1e-3) where {T}
-    bound, minimizers = minimise(x -> -f(x), X, tol)
+function maximise(f, X::T; str::Strategy=defstra ,tol=1e-3 ) where {T}
```

Comment:
Use full variable names: `str` sounds like a string.

Reply:
Yes you are right. I'll rename it to `structure`